src/Sarif.Sarifer/TextBufferViewTrackingInformation.cs

```diff
@@ -22,6 +23,7 @@ internal TextBufferViewTrackingInformation()
         {
             Views = new List<ITextView>();
             LogId = Guid.NewGuid().ToString();
+            CancellationToken = new CancellationToken();
```

Comment:
>CancellationToken = new CancellationToken(); [](start = 12, length = 44)

Here you would create a CancellationTokenSource. Usually you would use CancellationTokenSource<bool> since it doesn't support void (sadness).